src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1122,7 +1122,7 @@ void MainWindow::slot_open_project()
         get_open_filename(
             this,
             "Open...",
-            "Project Files (*.appleseed);;Packed Project Files (*.appleseedz);;All Files (*.*)",
+            "Project Files (*.appleseed *.appleseedz);;Plain Project Files (*.appleseed);;Packed Project Files (*.appleseedz);;All Files (*.*)",
```

Comment:
You're missing a ; between *.appleseed and *.appleseedz (check how other filters are defined).